src/dropbox.js

```diff
@@ -19,6 +19,10 @@ try {
   fetch = window.fetch.bind(window);
 }
 
+const b64 = typeof btoa === 'undefined'
+  ? (str) => Buffer.from(str).toString('base64')
+  : btoa;
+
```

Comment:
What's the reasoning behind this logic? Why not just use Buffer across the board?

Reply:
On that note, don't we need to include the Buffer dependency again?

Reply:
So this follows the conversation in PR [#259](https://github.com/dropbox/dropbox-sdk-js/pull/259#discussion_r421025252).  Essentially, we don't want to use buffer because it would require us to add in the licenses and whatnot.  Node by default includes buffers and the browsers by default include btoa.  So this essentially is the same thing we are doing with crypto and fetch.

Reply:
Okay, as long as there are no differing behaviors across browsers for btoa (that would cause issues here) this should be fine.